main.js

```diff
@@ -0,0 +1,194 @@
+
+let game
+let player1
+let player2
+const renderer = new Renderer()
+
+const moveToCell = function(currentCell, nextCell, direction){
+    let isBorderCell
+    switch(direction){
+        case 'right':
+            isBorderCell = game.isRightBorderCell(currentCell.y)
+            break;
+        case 'left':
+            isBorderCell = game.isLeftBorderCell(currentCell.y)
+            break;
+        case 'up':
+            isBorderCell = game.isTopBorderCell(currentCell.x)
+            break;
+        case 'down':
+            isBorderCell = game.isBottomBorderCell(currentCell.x)
+            break;
+        default:
+            isBorderCell = false
+    }
+    if(!(game.isWallCell(nextCell.x, nextCell.y) || game.isPlayerCell(nextCell.x, nextCell.y) || isBorderCell)){
+        let coins = 0
+        if(game.isCoinCell(nextCell.x, nextCell.y)){
+            coins = 10
+        }
+        game.alter(currentCell.x, currentCell.y, 'Empty')
+        game.alter(nextCell.x, nextCell.y, currentCell.type)
+        if(currentCell.type == 'Player1'){
+            switch(direction){
+                case 'right':
+                    player1.y +=1
+                    break;
+                case 'left':
+                    player1.y -=1
+                    break;
+                case 'up':
+                    player1.x -=1
+                    break;
+                case 'down':
+                    player1.x +=1
+                    break;
+                default:
+            }          
+            player1.coins+= coins
+        }else{
+            switch(direction){
+                case 'right':
+                    player2.y +=1
+                    break;
+                case 'left':
+                    player2.y -=1
+                    break;
+                case 'up':
+                    player2.x -=1
+                    break;
+                case 'down':
+                    player2.x +=1
+                    break;
+                default:
+            } 
+            player2.coins+= coins
```

Comment:
instead of repeating your code for player1 and player2, you could do something like this:
```
const player = currentCell.type == 'Player1' ? player1 : player2
switch(direction){
    case 'right':
        player.y +=1
        break;
    case 'left':
        player.y -=1
        break;
    case 'up':
        player.x -=1
        break;
    case 'down':
        player.x +=1
        break;
    default:
}          
player.coins+= coins
```